django/views/templates/default_urlconf.html

```diff
@@ -127,78 +117,54 @@
             border-top: 1px solid #efefef;
           }
           .option {
-            float: left;
-            width: 33.33%;
+            display: grid;
+            grid-template-columns: min-content 1fr;
+            gap: 10px;
             box-sizing: border-box;
-            padding-right: 5px;
+            text-decoration: none;
           }
           .option svg {
-            width: 25px;
-            height: 25px;
+            width: 1.5rem;
+            height: 1.5rem;
             fill: gray;
             border: 1px solid #d6d6d6;
             padding: 5px;
             border-radius: 100%;
-            float: left;
-            margin-right: 10px;
-          }
-          .option div {
-            display: table;
-          }
-          .option h4 {
-            color: #19865C;
-            font-size: 19px;
           }
           .option p {
             font-weight: 300;
-            font-size: 15px;
-            padding-top: 3px;
-            color: #757575;
+            line-height: 1.25;
+            color: #525252;
+          }
+          .option .option__heading {
+            color: #19865C;
+            font-size: 1.25rem;
+            font-weight: 400;
           }
           @media (max-width: 996px) {
             body, footer {
               max-width: 780px;
             }
-            .logo h2 {
-              margin-left: 0px;
-            }
           }
           @media (max-width: 800px) {
-            footer, main {
-              position: relative;
-            }
             footer {
               height: 100%;
-            }
-            .option {
+              grid-template-columns: 1fr;
+              gap: 60px;
```

Comment:
Apart from the RTL styles, this is also nicer than the `margin-bottom` it replaces because it won’t add extra whitespace at the end of the page after the last item.